api/v2.0/swagger.yaml

```diff
@@ -2044,23 +2055,23 @@ parameters:
     type: string
     required: false
     minLength: 1
+  isResourceName:   
+    name: X-Is-Resource-Name
+    description: The flag to indicate whether the parameter in the path is the name of the resource
```

Comment:
Could you please elaborate on the behavior here?  To specify that for some API the code will check if it's an ID or Name but if the header value is `true` the code will treat the parameter as a name directly

Reply:
More description added to this parameter.  

For this reason, a new parameter `X-Resource-Name-In-Location` is introduced to the APIs which can return both name and id in the Location response.